src/edfi-paging-test/edfi_paging_test/__main__.py

```diff
@@ -0,0 +1,20 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+import sys
+
+from helpers.argparser import parse_main_arguments  # type: ignore
+
+
+def main() -> None:
```

Comment:
The configargparse documentation implies that environment variables are automatically interpreted based on the "long name" of the setting. Thus for example I expect to be able to set env vars `BASEURL`, `KEY`, `SECRET`, etc.

In the LMS toolkit we made these names explicit. I did not remember that this was of necessity, rather than preference (for example, turning the `--server` argument into env var `DB_SERVER` because the word "server" is so generic as an environment variable).

So looks like we need to be explicit.

While investigating this, I also realized that we don't need to pass arguments around. Even though the configuration is in a separate module from `__main__`, it will still read `sys.argv` just fine.

Since my investigation caused me to make most of the necessary changes already, I'll tidy up and push a commit.